test/e2e/expirable-airdrop.spec.ts

```diff
@@ -289,4 +289,58 @@ describe('Expirable airdrop', () => {
       });
     });
   });
+
+  describe('updateMerkleRoot()', () => {
+    let newMerkleRoot: Uint8Array;
+
+    given(async () => {
+      newMerkleRoot = ethers.utils.randomBytes(32);
+    });
+
+    when('caller is not governor', () => {
+      then('revert with custom error', async () => {
+        await expect(expirableAirdrop.connect(alice).updateMerkleRoot(newMerkleRoot)).to.be.revertedWithCustomError(
+          expirableAirdrop,
+          'OnlyGovernor'
+        );
+      });
+    });
+
+    when('caller is governor', () => {
+      given(async () => {
+        await expirableAirdrop.connect(deployer).updateMerkleRoot(newMerkleRoot);
+      });
+
+      then('merkle root should have the new value', async () => {
```

Comment:
Could we also check that event is emitted?

Reply:
Sure!